native-auth/resources.md

```diff
@@ -0,0 +1,36 @@
+## Resources
+
+```
+dto NativeAuthDecoded:
+    ttl: number;
+    origin: string;
+    address: Address;
+    signature: string;
+    block_hash: string;
+    body: string;
+    extra_info?: Any;
+```
+
+```
+dto NativeAuthResult:
```

Comment:
Is this used?

Reply:
Renamed it to `NativeAuthValidateResult`.